.github/workflows/cdktf-provider-docs-rollout.yml

```
@@ -0,0 +1,61 @@
+name: CDKTF Provider Docs Rollout
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: 0 0 * * 1
```

I'm bad at CRON and had to Google

```suggestion
    - cron: 0 0 * * 1 # Mondays at midnight
```